sample/test.py

```diff
@@ -31,11 +26,20 @@ def run(self):
     def __init__(self):
         logging.basicConfig()
         self.logger = logging.getLogger(__name__)
-        #signal.signal(signal.SIGINT, signal.SIG_IGN)
         logging.getLogger('mesoshttp').setLevel(logging.DEBUG)
         self.driver = None
-        self.client = MesosClient(mesos_urls=['http://127.0.0.1:5050'])
-        #self.client = MesosClient(mesos_urls=['zk://127.0.0.1:2181/mesos'])
+        # By default use ZK for master detection
+        self.client = MesosClient(mesos_urls=['zk://leader.mesos:2181/mesos'])
```

Comment:
you suppose here that leader.mesos is known.... not if service is not in DC/OS

Reply:
Good call. I'll correct.

Reply:
as this is for testing only, could be managed via test params or env variables

Reply:
Good suggestion. I've added the ability to set via environment variable.